### ISSUE TITLE
Hi, I got a bad reconstruction result of mesh from the reference network

### ISSUE THREAD

**Issue by neverstopZyy** [tarun738/i3DMM#5]
I selected the "10084_e17" model from the test set you provided, and tried to use it to pretrain the reference network as your paper. However, when I try to reconstruct the training result, I found it's terribly bad. Could you please tell me what's wrong with it?
Here is my result:
![image](https://user-images.githubusercontent.com/42933584/140844865-f846320b-9c0e-44fa-abf9-8261ea86173a.png)
And the ground truth:
![image](https://user-images.githubusercontent.com/42933584/140845167-cd752f4c-445a-4ab8-aab5-44528025e200.png)

I use these three command:
1. prepare the training data set
python preprocessData.py --samples_directory ./data --input_meshes_directory ../dataset  -e headModel -s Train
2. train the model
python train_i3DMM.py -e headModel -s ./headModel/splits/referenceShapeSample.json -d data -r 1
3. reconstruct the reference network result 
python fit_i3DMM_to_mesh.py  -e headModel -s ./headModel/splits/referenceShapeSample.json -d data -r 1

**Comment by tarun738** [comment]
Hello!

Thank you for your interest in our paper!

This is expected since the data is noisy. You need to train the entire model and not just the reference when over fitting. Also, since you're over fitting to a mesh, you need to train for more epochs. (Around 5-10k, generally takes an hour, see you the other issue that was just closed)

We only use the trained reference as an initialization. We also learn the reference while training the entire network.

Hope that answers.

Best,
Tarun
